src/Evolution/Systems/GrMhd/ValenciaDivClean/NewmanHamlin.cpp

```diff
@@ -59,7 +57,8 @@ boost::optional<PrimitiveRecoveryData> NewmanHamlin::apply(
   size_t valid_entries_in_aitken_pressure = 1;
   bool converged = false;
 
-  while (true) {  // will break when relative pressure change is < 1e-10
+  while (true) {  // will break when relative pressure change is <
+                  // relative_tolernance_
     if (max_iterations_ == iteration_step and not converged) {
```

Comment:
`UNLIKELY` (I have a commit that does this, but if you want to add it that would be nice :) )